options.html

```diff
@@ -113,6 +113,18 @@ <h1 class="text-container py-2 px-2"><i class="fas fa-globe"></i> Global</h1>
                         </div>
                     </div>
                 </div>
+                <div class="form-group text-container p-2">
+                    <label class="font-weight-bold">Update on Follows page <a data-default="updateOnFollows" class="btn btn-sm btn-secondary"><i class="fas fa-trash"></i><span class="d-none d-xl-inline"> Restore default</span></a></label>
+                    <p class="d-none d-xl-block">Automatically check if new chapters have already been read (on MyAnimeList). Only checks the first 7 with unread higher chapters.</p>
+                    <div data-option="updateOnFollows" data-type="checkbox" class="col px-0 my-auto input-group">
+                        <div class="input-group-prepend">
+                            <button type="button" class="btn btn-success input-prepend col px-2">Enabled</button>
+                        </div>
+                        <div class="input-group-append">
+                            <button type="button" class="btn btn-secondary input-append col px-2 active">Disabled</button>
```

Comment:
Why always make the disabled active?

Reply:
It's just a forgotten class when I made the checkboxes a long time ago, it does nothing.

Reply:
It doesn't do nothing, it makes the background slightly darker
![image](https://user-images.githubusercontent.com/59023762/80002794-6584a300-84c0-11ea-9c22-ebd15c7df8b9.png)

Reply:
Yes, but what I mean is that it does nothing for the options, it's only visual.